task/buildah-remote-oci-ta/0.1/buildah-remote-oci-ta.yaml

```diff
@@ -363,8 +365,8 @@ spec:
       buildah images
       buildah tag localhost/rhtap-final-image "$IMAGE"
       container=$(buildah from --pull-never "$IMAGE")
-      buildah mount "$container" | tee /var/workdir/container_path
-      echo $container > /var/workdir/container_name
+      buildah mount "$container" | tee /workspace/container_path
+      echo $container > /workspace/container_name
```

Comment:
This one uses `syft dir:$(cat /var/workdir/container_path) --output cyclonedx-json=/var/workdir/sbom-image.json` though

Reply:
Ugh, I think these both need a change.

The /workspace thing is a tekton implementation detail, we should not be relying on it as it is not documented AFAIK. 

I will change both of these to use a /shared volume mount.